src/components/wallet/methods/updateAccount.ts

```diff
@@ -29,7 +29,7 @@ export default async function updateAccount(this: Wallet) {
             'paging_token',
           ]),
         }
-        this.logger.response(selfURL.self.href, account)
+        // this.logger.response(selfURL.self.href, account)
```

Comment:
I want to comment these out until we get the foldable JSON just so we don't start off with a lot of noise